src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap

```diff
@@ -15,145 +15,61 @@ exports[`PodcastExternalLinks Should render external links 1`] = `
   }
 }
 
-.emotion-3 {
-  position: relative;
-  z-index: 0;
-  color: #3F3F42;
-  margin-top: 2rem;
-  margin: 0;
-  margin-top: 1rem;
-}
-
-@media (min-width: 37.5rem) {
-  .emotion-3 {
-    margin-top: 1.5rem;
-  }
-}
-
-@media (min-width: 63rem) {
-  .emotion-3 {
-    margin-bottom: 1.5rem;
-  }
-}
-
-@media (min-width: 25rem) {
-  .emotion-3 {
-    margin-top: 1rem;
-    margin-bottom: 0;
-  }
-}
-
-@media (min-width: 37.5rem) {
-  .emotion-3 {
-    margin-top: 1rem;
-    margin-bottom: 0;
-  }
-}
-
-.emotion-5 {
-  margin: 0;
-  padding: 0;
-}
-
-.emotion-7 {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-flex-direction: column;
-  -ms-flex-direction: column;
-  flex-direction: column;
-}
-
-.emotion-9 {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-box-flex-flow: row nowrap;
-  -webkit-flex-flow: row nowrap;
-  -ms-flex-flow: row nowrap;
-  flex-flow: row nowrap;
-  -webkit-box-pack: justify;
-  -webkit-justify-content: space-between;
-  justify-content: space-between;
-  min-height: 2.75rem;
-  -webkit-align-items: baseline;
-  -webkit-box-align: baseline;
-  -ms-flex-align: baseline;
-  align-items: baseline;
-}
-
-@media (min-width: 37.5rem) {
-  .emotion-9 {
-    -webkit-align-items: stretch;
-    -webkit-box-align: stretch;
-    -ms-flex-align: stretch;
-    align-items: stretch;
-  }
-}
-
-.emotion-11 {
+.emotion-2 {
   font-size: 1.125rem;
   line-height: 1.375rem;
   font-family: ReithSans,Helvetica,Arial,sans-serif;
   font-weight: 400;
   font-style: normal;
-  background-color: #FDFDFD;
-  margin: 1rem 0;
-  padding-right: 0.5rem;
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  -webkit-align-items: center;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
+  color: #222222;
```

Comment:
We shouldn't be using the link colour as this isn't a link

Reply:
Thank you for the review @greenc05. It looks from the Zeplin designs that the color is `#222222` - See https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5fbc0b18dbfa4a5b7b118549

Reply:
Thanks for updating as per Emily's confirmation